win32/ogg.def

```diff
@@ -2,7 +2,6 @@
 ; ogg.def
 ; List of exported functions for Windows builds.
 ;
-LIBRARY
```

Comment:
The docs say [a LIBRARY line is required](https://docs.microsoft.com/en-us/cpp/build/exporting-from-a-dll-using-def-files?view=vs-2017). Would `LIBRARY ogg` work? Is this an ABI change either way?